frontend/packages/knative-plugin/src/components/overview/__tests__/KnativeOverview.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as _ from 'lodash';
+import { PodRing } from '@console/shared';
+import { ResourceSummary } from '@console/internal/components/utils';
+import { KnativeOverview } from '../KnativeOverview';
+import { mockOverviewItemData } from './overview-knative-test-data';
+
+describe('KnativeOverview', () => {
+  it('should render PodRing if obj.kind is RevisionModel.kind', () => {
+    const wrapper = shallow(<KnativeOverview item={mockOverviewItemData.item} />);
```

Comment:
as you are using `mockOverviewItemData.item` can we reuse revisions data which we have?

Reply:
revisions data from the topology can be used for `item.obj` only